client/src/Movies/Movie.js

```diff
@@ -1,25 +1,23 @@
-import React, { useState, useEffect } from 'react';
-import axios from 'axios';
+import React, { useState, useEffect } from "react";
+import axios from "axios";
 
-const Movie = (props) => {
+const Movie = props => {
   const [movie, setMovie] = useState({});
- 
+
   useEffect(() => {
-    const id = 1;
+    const id = props.match.params.id;
```

Comment:
Looks great. Pop Quiz: Where is "match" coming from?

Reply:
`BrowserRouter` or as `Router` (by convention) component that wraps around `<App />`